silk-react-components/src/HierarchicalMapping/Components/MappingsHeader.jsx

```diff
@@ -79,67 +63,72 @@ const MappingsHeader = React.createClass({
         ) : (
             false
         );
-
+        
         const self = this;
```

Comment:
`self` not needed. `this` can be used directly.

Reply:
this is rly something new for me 😂